search/includes/classes/queue/class-cron.php

```diff
@@ -114,9 +125,47 @@ public function process_jobs( $job_ids ) {
 		$this->queue->process_jobs( $jobs );
 	}
 
+	/**
+	 * Given a term taxonomy id, queue all posts for reindexing that match it
+	 *
+	 * @param {int} $term_taxonomy_id The term taxonomy id you want to index
+	 */
+	public function queue_posts_for_term_taxonomy_id( $term_taxonomy_id ) {
+		$args = array(
+			'posts_per_page' => self::TERM_UPDATE_BATCH_SIZE,
+			'paged' => 1,
```

Comment:
We can optimize this query a little...

```suggestion
			'paged' => 1,
			'fields' => 'ids',
			'update_post_meta_cache' => false,
			'update_post_term_cache' => false,
```